recipe/meta.yaml

```diff
@@ -10,21 +10,16 @@ source:
   sha256: 36051ac530ddb461a86b6227c4b09d95f30a1d1043de2b4a592e97ae8a84fcdf
 
 build:
-  skip: true  # [win or py<35]
```

Comment:
Do we still need to skip Windows?

Reply:
No, I missed that one.

Reply:
No worries. We could just add `__unix` to `run`